code-challenges.test.js

```diff
@@ -24,20 +63,46 @@
 
 // b) Create the function that makes the test pass.
 
+// Done. See above :)
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
+/**
+ * Return an array of odds by using filter() and then chain the conditional to check if it's a number
+ * Once you have that, call sort to sort from least to greatest
+ */
+
+const onlyOdds = (arr) => {
+  const odds = arr.filter(number => Number.isInteger(number) && !(number % 2 === 0));
+  return odds.sort((a, b) => a - b);
+};
+
 // a) Create a test with expect statements for each of the variables provided.
 
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+
+// console.log(onlyOdds(fullArr1));
+
 // Expected output: [-9, 7, 9, 199]
 
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
+
 // Expected output: [-823, 7, 23]
 
+// console.log(onlyOdds(fullArr2));
 
 // b) Create the function that makes the test pass.
 
+describe('onlyOdds()', () => {
+  it('should return [-9, 7, 9, 199] when given fullArr1', () => {
```

Comment:
Approaching your `it` statements differently in the future might be a good idea. The goal is to describe the expected behavior of a function. You are using your test variables which doesn't actually describe what the function is supposed to do.